### DISCUSSION TITLE
At what point are Anvil Labs features "safe" to use?

### DISCUSSION THREAD

**Discussion by jshaffstall** [anvilistas/anvil-labs#52] [Q&A] [answered]
Asking because some of the non-blocking features, especially the async server calls, would be super useful in a portion of my current project.  At the moment I'm hiding certain server calls in user interaction dead space, but being able to make them async would be great.  

But, this is a project that is fairly close to being production ready.  I wasn't sure if the general rule was "don't use anything from Labs until it migrates to Extras", or "Some features are safer than others".

**Comment by meatballs** [comment, marked as answer]
When we are properly confident (that they work, that we've tested them, that the API is how we want it, that the docs are decent), they'll be migrated into anvil extras.

**Reply by meatballs** [reply]
But, it's all open source, so you're free to take whatever you want and copy it to your app (as long as you include the licence and copyright).
